### PR TITLE
Add netboxr tool (beta version) - second attempt

### PR DESCRIPTION
**Pull request by sajo25**
Changes made based on results from automated [tests](https://github.com/bgruening/galaxytools/pull/1229).

### CODE REVIEW

File: tools/netboxr/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+categories: []
```
Comment (bgruening) [inline tools/netboxr/.shed.yml:1]: you need to pick one from https://toolshed.g2.bx.psu.edu/

File: tools/netboxr/.shed.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+description: netboxr_tool
```
Comment (bgruening) [inline tools/netboxr/.shed.yml:2]: better description would be nice :)

File: tools/netboxr/netboxr_xml.xml [hunk 3]
```diff
@@ -0,0 +1,182 @@
+<tool id="netboxr_tool" name="NetBox" version="1.6.0">
```
Comment (bgruening) [inline tools/netboxr/netboxr_xml.xml:1]: I would remove the _tool from the ID

File: tools/netboxr/netboxr_xml.xml [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+<!-- A simple description of the tool that will appear in the tool
+panel in Galaxy. -->
+<description> netboxr enables automated discovery of biological process
+modules by network analysis.</description>
+<!-- Handles exit codes in Galaxy. -->
+<requirements>
+<requirement type="package">r-optparse</requirement>
+<requirement type="package" version="1.6.0">bioconductor-netboxr</requirement>
+</requirements>
+<stdio>
```
Comment (bgruening) [inline tools/netboxr/netboxr_xml.xml:11]: remove stdio and use detect_errors see https://docs.galaxyproject.org/en/latest/dev/schema.html#error-detection

File: tools/netboxr/netboxr_xml.xml [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+--cutoff $cutoff --community $community
+ --globalModel $globalModel --globalIterations $globalIterations --globalNumber $globalNumber
+ --localModel $localModel --localIterations $localIterations --networkPlot $networkPlot --plotWidth $plotWidth
+ --outputSIF $outputSIF --neighborList $neighborList --modmem $modmem --nt $nt
+]]>
+<!-- add resolutionParam $resolutionParam after communityMethod
+after you make pull request-->
+</command>
+<inputs>
+<param name="gene_list" type="data" format=".txt" label="Gene list" help="A
```
Comment (bgruening) [inline tools/netboxr/netboxr_xml.xml:25]: can you follow maybe the IUC formatting of XML files to make it more readable?

File: tools/netboxr/netboxr_xml.xml [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+newline-delimited text file that contains a list of genes of interest"/>
+<param name="cutoff" type="text" label="P-Value Cut-Off"
```
Comment (bgruening) [inline tools/netboxr/netboxr_xml.xml:27]: that looks like type="float" and you can set a value="0.05"

**Comment by sajo25** [comment]
@bgruening Thanks for the feedback!

**Comment by bgruening** [comment]
@sajo25 anything we can help here?

**Comment by sajo25** [comment]
@bgruening I made the requested changes and created another [PR](https://github.com/bgruening/galaxytools/pull/1233). My understanding is that this PR needs to be merged, after which the tool will automatically be uploaded to the ToolShed. Is that right, or am I missing something?

**Comment by bgruening** [comment]
Ah didn't saw the new one, thanks.

**Pull request closed without merging**